topics/activitypub/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: ActivityPub
+github_url: https://github.com/w3c/activitypub
+logo: activitypub.png
+related: decentralized, federated, protocol, social-network, w3c
+short_description: Decentralized social networking protocol.
+topic: activitypub
+url: https://activitypub.rocks/
```

Comment:
I see on the w3c/activitypub repo, the web site is listed as http://w3c.github.io/activitypub/. However, I see some of the links on http://w3c.github.io/activitypub/ point to the activitypub.rocks domain, so I think this `url:` is fine as the activitypub.rocks site seems more user-friendly and provides a good introduction.